ansible/roles/rke/vars/main.yml

```diff
@@ -1,5 +1,7 @@
 ---
 kubectl_version: v1.15.2
+helm_version: v2.14.3
+rancher_version: v2.2.0
```

Comment:
Is there a way to share this with `roles/rancher/vars/main.yml`?

Reply:
Yeah, it's possible to define the version variable inside the `prod.yml` playbook. But would that variable actually be used in the `rancher` role?

Reply:
Didn't realize CLI versions weren't released in tandem, makes sense.

Reply:
I wanted to keep `vars` specific to each role inside the role's definition instead of defining them globally. Do we anticipate playbooks that use only a few roles but not the others or are we certain that we're going to run all roles all the time?